packages/components/pie-assistive-text/test/component/pie-assistive-text.spec.ts

```diff
@@ -1,28 +1,45 @@
-import { test, expect } from '@sand4rt/experimental-ct-web';
-import { PieAssistiveText } from '../../src/index.ts';
-
-const componentSelector = '[data-test-id="pie-assistive-text"]';
+import { test, expect } from '@playwright/test';
+import { BasePage } from '@justeattakeaway/pie-webc-testing/src/helpers/page-object/base-page.ts';
+import { assistiveText } from '../helpers/page-object/selectors.ts';
+import type { AssistiveTextProps } from '../../src/index.ts';
 
 test.describe('PieAssistiveText - Component tests', () => {
-    test('should render successfully', async ({ mount, page }) => {
-        // Arrange
-        await mount(PieAssistiveText);
+    test('should render successfully', async ({ page }) => {
+    // Arrange
+        const assistiveTextPage = new BasePage(page, 'assistive-text--default');
+        await assistiveTextPage.load();
 
         // Act
-        const assistiveText = page.locator(componentSelector);
+        const assistiveTextComponent = page.getByTestId(assistiveText.selectors.container.dataTestId);
 
         // Assert
-        expect(assistiveText).toBeVisible();
+        expect(assistiveTextComponent).toBeVisible();
     });
 
-    test('should set `variant` to `default` by default', async ({ mount }) => {
-        // Arrange
-        const component = await mount(PieAssistiveText);
+    test('should set `variant` to `default` by default', async ({ page }) => {
+    // Arrange
+        const assistiveTextPage = new BasePage(page, 'assistive-text--default');
+        await assistiveTextPage.load();
 
         // Act
-        const assistiveTextComponent = component.locator(componentSelector);
+        const assistiveTextComponent = page.getByTestId(assistiveText.selectors.container.dataTestId);
 
         // Assert
         await expect(assistiveTextComponent).toHaveClass(/c-assistiveText--default/);
     });
+
+    test('should have the `c-assistiveText--isVisuallyHidden` class applied when `isVisuallyHidden` is `true`', async ({ page }) => {
+        // Arrange
+        const props : AssistiveTextProps = {
+            isVisuallyHidden: true,
+        };
+        const assistiveTextPage = new BasePage(page, 'assistive-text--default');
+        await assistiveTextPage.load({ ...props });
+
+        // Act
+        const assistiveTextComponent = page.getByTestId(assistiveText.selectors.container.dataTestId);
+
+        // Assert
+        await expect(assistiveTextComponent).toHaveClass(/c-assistiveText--isVisuallyHidden/);
```

Comment:
My only concern here is that if I made an scss change to make the visually hidden class no longer hidden, would we know?

Reply:
Have added additional assertions to check for the expected CSS properties which should remedy this!

Reply:
Moved this over to be a visual test!